pkg/controllers/canary_controller.go

```diff
@@ -225,3 +243,27 @@ func (r *CanaryReconciler) includeNamespace(namespace string) bool {
 	}
 	return false
 }
+
+func getCheckIDsForCanary(canaryID uuid.UUID) []string {
+	checks, _ := db.GetAllChecksForCanary(canaryID)
```

Comment:
no error handling required ?

Reply:
Even in case, the error happens, we won't be stopping the reconcile loop or doing anything here. So I thought would be better to just ignore it